app/graphql/resolvers/webhooks_resolver.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class WebhooksResolver < Resolvers::BaseResolver
+    include AuthenticableApiUser
+    include RequiredOrganization
+
+    description 'Query Webhooks'
+
+    argument :page, Integer, required: false
+    argument :limit, Integer, required: false
+    argument :status, Types::Webhooks::StatusEnum, required: false
+
+    type Types::Webhooks::Object.collection_type, null: false
+
+    def resolve(page: nil, limit: nil, status: nil)
+      validate_organization!
+
+      webhooks = current_organization.webhooks
+        .page(page)
+        .per(limit)
+
+      webhooks = webhooks.where(status:) if status.present?
+
+      webhooks.order(updated_at: :desc)
```

Comment:
Are we sure we want `updated_at` here? Not `last_retried_at` or `created_at`?

Reply:
since the `last_retried_at` can be `nil`, I think the source of truth is the `updated_at`, wdyt?

Reply:
works for me - will see in the future if not :)